.github/workflows/test-containers.yml

```diff
@@ -0,0 +1,42 @@
+name: test test_containers
+on:
+  workflow_call:
+  push:
+  pull_request:
+jobs:
+  test:
+    strategy:
+      matrix:
+        platform:
+          - ubuntu-latest
+    runs-on: ${{ matrix.platform }}
```

Comment:
nit
```suggestion
    runs-on: ubuntu-latest
```

Reply:
eventually we want to test on mac, which won't work yet. but ill remove this for now